client/src/App.js

```diff
@@ -1,26 +1,70 @@
-import React, { Component } from 'react'
-import 'semantic-ui-css/semantic.min.css'
-import './App.css'
-import { BrowserRouter as Router, Route } from 'react-router-dom'
+import React, { Component } from "react";
+import { BrowserRouter as Router, Route } from "react-router-dom";
+import axios from "axios";
+import config from "./config";
+import Home from "./Home";
+import Login from "./components/login";
+import Team from "./components/team";
+import Tournaments from "./components/tournaments";
+import "semantic-ui-css/semantic.min.css";
+import "./App.css";
 
-import Home from './components/home'
-import Login from './components/login'
-import Team from './components/team'
-import Tournaments from './components/tournaments'
+const userManager = Clz => {
+  return class extends Component {
+    state = {
+      loading: true,
+      loggedIn: null,
+      error: null,
+      user: {}
+    };
+
+    componentDidMount = async () => {
+      try {
+        const user = await axios.get(`${config.SERVER}/self`, {
```

Comment:
axios better then fetch?

Reply:
Rob already hooked it up so I just went with it